scripts/bytecodecompare/prepare_report.py

```diff
@@ -422,6 +436,13 @@ def commandline_parser() -> ArgumentParser:
         choices=[c.value for c in CompilerInterface],
         help="Compiler interface to use.",
     )
+    parser.add_argument(
+        '--execution-arch',
+        dest='execution_arch',
+        default=ExecutionArchitecture.NATIVE.value,
+        choices=[c.value for c in ExecutionArchitecture],
+        help="Select the architecture of the universal binary that should be executed. (Only relevant for macOS)",
```

Comment:
Listing the actual options would be helpful. Also, not a big fan of `native` vs `native-x86-64` nomenclature, I'd rather if it was just `arm64` and `x86-64`.

Reply:
Or even just `arm` and `x86`, who even uses 32 bit addressing now days.

Reply:
not sure what you mean.. but I agree that `native` vs `native-<something-else>` is confusing.. so I just call that `x86-64` and `x86-64h` there

Reply:
Wait, `native` is just `arm` right? So `arm` and `x86` should be sufficient - no need for the `64`.

Reply:
Yeah, I guess I could just change that

Reply:
there is now only `native` and `osx_intel`